README.md

```diff
@@ -10,5 +10,18 @@ Bob,Alice,black       | Bob was black, Alice white, and Alice won
 Alice,Bob,tie         | Alice was white, Bob black, and they tied
 </pre>
 
-### How to view elo chart
+## How to view elo chart
 Go to https://chesselo.netlify.com/
+
+## Developing
+
+```sh
+# Run dev server
+npm run serve
+
+# Run tests
+npm test
+
+# Production build
+npm run build
+```
```

Comment:
Dev usage notes

Reply:
When would I need to run `npm run build`?

Reply:
Probably never locally. Netlify runs it